moda-contracts/src/interfaces/Releases/IReleases.sol

```diff
@@ -6,14 +6,76 @@ pragma solidity 0.8.21;
  * @dev Interface for Releases
  */
 interface IReleases {
+    event ReleaseCreated(uint256 tokenId);
+    event ReleaseWithdrawn(address indexed receiver, uint256 tokenId, uint256 amount);
+
+    /**
+     * @dev Initializes the contract
+     * @param admin - The address that will be given the role of default admin. See {AccessControl}
+     * @param releaseAdmins - An array of addresses that will be given the role of release admin
+     * @param name_ - The name of the Releases contract
+     * @param symbol_ - The symbol of the Releases contract
+     * @param catalog - The address of the catalog contract
+     * @param splitsFactory - The address of the moda splits contract
+     */
+    function initialize(
+        address admin,
+        address[] memory releaseAdmins,
+        string memory name_,
+        string memory symbol_,
+        address catalog,
+        address splitsFactory
+    ) external;
+
+    /**
+     * @dev Creates a new release. If the contract is switched to open
+     * an extra check is made to ensure the caller has access to the track.
+     *
+     * @notice Only release admins can call this function
+     * @param receiver - The address that will receive the release token
+     * @param royaltyAmount - The percentage of sale prices
+     * that should be paid to the beneficiary for re-sales.
+     *  Calculated by <NOMINATOR> / 10,000. e.g. For 10% royalties, pass in 100
+     * @param beneficiary - The address that will receive resale royalties
+     * @param uri_ - The URI of the token metadata
+     * @param trackIds - An array containing the registered track ids of the tracks
```

Comment:
Maybe add a note here that the track must be registered in the Catalog. Help out users to understand the contract relationships